src/index.html

```diff
@@ -1,13 +1,46 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <div class="card__img">
+        <img
+          class="card__item"
+          src="images/imac.jpeg"
+          alt="card-item">
+      </div>
```

Comment:
You should add an alt that describes the image.

```suggestion
      <div class="card__img-wrp">
        <img
          class="card__img"
          src="images/imac.jpeg"
          alt="Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)"
        >
      </div>
```